hack/jenkins/build.sh

```diff
@@ -47,7 +47,9 @@ docker images -q | xargs -r docker rmi
 # Build
 go run ./hack/e2e.go -v --build
 
-# Push to GCS
-./build/push-ci-build.sh
+[[ ${KUBE_SKIP_PUSH_GCS:-} =~ ^[yY]$ ]] || {
```

Comment:
The more common pattern seems to be looking for "true" - any reason to prefer [yY] here?